admin-dev/themes/new-theme/js/nav_bar.js

```diff
@@ -91,6 +90,9 @@ export default class NavBar {
         '.menu-collapse',
         function onNavBarClick() {
           $('body').toggleClass('page-sidebar-closed');
+          setTimeout(() => {
+            $('.main-menu').toggleClass('sidebar-closed');
+          }, 400)
```

Comment:
Do you think it would be possible to have a callback mechanism here instead of a setTimeout ?

Reply:
The callback mech wouldn't replace the setTimeout because sidebar-closed toggleClass needs to be executed 400ms after the first toggleClass, but you should be right the correct solution would be to include that setTimeout inside a promise resolve so it doesnt get executed if body doesnt exist, but do we really need to care if body isnt existing ?

Reply:
So if I get it right the 400ms are the time needed for the css animation to be executed.
I guess there's no other way (except to remake the whole animation system). 👍

Reply:
`transition: all .5s ease-out;`

Thats set in the transition of .nav-bar yes, the substraction of 100ms is to show the text a bit before the animation's end